docs/sequenceDiagrams/ListSequenceDiagramCode.txt

```diff
@@ -0,0 +1,27 @@
+title List
+
+ListCommand->*+Timetable: <<create>>
+Timetable-->ListCommand:
+ListCommand->*+ForecastFilter: <<create>>
+ForecastFilter-->-ListCommand:
+ListCommand->*+TaskFilter: <<create>>
+TaskFilter-->-ListCommand:
+
+
+Timetable->+TaskList: getFilteredList()
+TaskList->+TaskFilter: isTaskIncluded()
+TaskFilter-->-TaskList:
+TaskList->+ForecastFilter: removeExcludedDates()
+ForecastFilter-->-TaskList:
+
+TaskList->TaskList: <<create>>
+activate TaskList
+deactivate TaskList
+
+TaskList-->-Timetable:
+
+Timetable->Timetable: populateTimetable()
+activate Timetable
+deactivate Timetable
+
+Timetable-->-ListCommand:
```

Comment:
Basic functions are covered that's good. Cleaned up the diagram a little bit, check it out and tell me what you think


title List

actor ATHENA

ATHENA->ListCommand: execute()
activate ListCommand

ListCommand->*ImportanceFilter:
activate ImportanceFilter
ImportanceFilter-->ListCommand:
destroy ImportanceFilter

ListCommand->*ForecastFilter: 
activate ForecastFilter
ForecastFilter-->ListCommand:
deactivate ForecastFilter


ListCommand->*Timetable:
activate Timetable

Timetable->+TaskList: getFilteredList()
TaskList->+TaskFilter: isTaskIncluded()
TaskFilter-->-TaskList:
TaskList->+ForecastFilter: removeExcludedDates()
ForecastFilter-->-TaskList:

destroy ForecastFilter
destroy TaskFilter



TaskList-->-Timetable:

Timetable->Timetable: populateTimetable()
activate Timetable
deactivate Timetable

Timetable-->-ListCommand:
ListCommand-->-ATHENA: